src/modules/fancyzones/editor/FancyZonesEditor/GridEditorWindow.xaml

```diff
@@ -37,7 +37,8 @@
         </Grid>
         <StackPanel Margin="16">
             <StackPanel Margin="0,8,0,0">
-                <TextBlock 
+                <TextBlock
+                    Focusable="True"
```

Comment:
while testing, keyboard focus doesn't really land on these TextBlocks, so are these Focusable="True" really needed, as AutomationProperties.Name is set on GridZone bellow

Reply:
It's supposed to land there the first time you open the Window. I think you need to do Ctrl+Tab to get back to that Windows with the textboxes though. They'll focus after the buttons, though.